src/esp/sensor/Sensor.h

```diff
@@ -44,22 +44,12 @@ enum class SensorSubType {
 
 // Specifies the configuration parameters of a sensor
 struct SensorSpec {
```

Comment:
Any feedback on if this SensorSpec looks as expected would be great, or if there is anything else to move.